shared/js/smart-screen/scrollable.js

```diff
@@ -203,7 +203,6 @@
       // Determine initial focus (depends on reversed or not)
       var initNode = this.getItemFromNode(
                      this.getNode(this._isReversed ? this.length - 1 : 0));
-      this.silentFocus(initNode);
 
```

Comment:
The purpose of this line is, when we show the list reversely, the rightmost item should be focused.
And deleting this would break this feature.